src/data/navigation/sections/rest.js

```diff
@@ -8,13 +8,31 @@ module.exports = [
         path: "/rest/quick-reference/",
     },
     {
-        title: "Generate a local API Reference",
-        path: "/rest/quick-reference/generate-local.md",
+        title: "Authentication",
+        path: "/rest/authentication/",
+        pages: [
+            {
```

Comment:
I think you should add a TOC entry for `/rest/authentication` as a page named "Overview", rather than relying on the expanding link, because i think those are inobvious sometimes.